flink-core/src/test/java/org/apache/flink/util/FileUtilsTest.java

```diff
@@ -226,37 +226,11 @@ public void testDeleteDirectoryConcurrently() throws Exception {
 
 	@Test
 	public void testCompression() throws IOException {
```

Comment:
nit: rename to `testCompressionOnAbsolutePath` for comparing with below `testCompressionOnRelativePath`. It is also not mandatory.